custom/icds_reports/sqldata/exports/demographics.py

```diff
@@ -50,6 +50,7 @@ def order_by(self):
         order_by = []
         for column in order_by_columns:
             order_by.append(OrderBy(column.slug))
+        order_by.append(OrderBy('aggregation_level'))
```

Comment:
I think including this in line 50 with order_by's definition would result in the same result with one less line of code (theoretically simpler) ?

Reply:
it could but then the location based ordering would be messed up